### PR TITLE
first draft of service-related articles

### PR DESCRIPTION
**Pull request by frconil**
# What changed, and why it matters

As discussed earlier, I've added articles about key operations that can be done for Aiven services. This includes as much as possible a step by step guide with the Aiven console, and its equivalent using the CLI. 

Thoughts and feedback welcome :)

### CODE REVIEW

**Comment by laysauchoa** [comment]
Hi @frconil! Thanks for your contribution. 

Please, would be possible to run `make spell` and fix some spellings? thanks

File: .github/styles/Vocab/Docs/accept.txt [hunk 1]
```diff
@@ -48,6 +48,7 @@ httpie
 hyperloglogs
 hypertable
 Instana
+ip
```
Comment (laysauchoa) [inline .github/styles/Vocab/Docs/accept.txt:51]: IP is acceptable already, so maybe try to replace `ip` with `IP` first. You'll not need to add here.

Comment (frconil) [reply .github/styles/Vocab/Docs/accept.txt:51]: in this case, make spell is blocking on the use of lowercase IP in my example (`example_ip`)

Comment (lornajane) [reply .github/styles/Vocab/Docs/accept.txt]: Make sure that you use literal/code markup like you did in this comment, also in your text. The spell checker doesn't try to apply its stuff to words.

Comment (frconil) [reply .github/styles/Vocab/Docs/accept.txt]: yeah, I assumed that's what the double backticks would do, but Laysa showed me another way to achieve that result. Thanks for the heads up!

File: docs/platform/howto/scale-services.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Scale a service using the Aiven client (CLI)
+--------------------------------------------
+
+1. Prepare the command to scale the service.
+
+2. Add the ``service_to_scale`` parameter to specify the service you want to scale, and the ``service_plan`` for the target plan. For example, if you want to scale your service to ``service_plan``, the command would be something like:
+
+    avn service update --plan service_plan service_to_scale
+
+You can follow the progress of your changes with:
```
Comment (laysauchoa) [inline docs/platform/howto/scale-services.rst:28]:
```suggestion
You can follow the progress of your changes with::
```

File: docs/platform/howto/monitoring-services.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+1. Prepare the command to review the service.
+
+2. Add the ``service_to_review`` parameter to specify the service you want to review.
+
+3. To review your service logs, you can use this command:
+
+    avn service logs ``service_to_review``
+
+3. To review your service metrics, you can use this command:
```
Comment (laysauchoa) [inline docs/platform/howto/monitoring-services.rst:26]:
```suggestion
3. To review your service metrics, you can use this command::
```

File: docs/platform/howto/monitoring-services.rst [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+
+Review a service using the Aiven client (CLI)
+----------------------------------------------
+
+1. Prepare the command to review the service.
+
+2. Add the ``service_to_review`` parameter to specify the service you want to review.
+
+3. To review your service logs, you can use this command:
```
Comment (laysauchoa) [inline docs/platform/howto/monitoring-services.rst:22]:
```suggestion
3. To review your service logs, you can use this command::
```

File: docs/platform/howto/migrate-services.rst [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+a. You can retrieve the list of available clouds and regions with:
+
+    avn cloud list
+
+2. Add the ``service_to_migrate`` parameter to specify the service you want to migrate, and the ``cloud_region`` for the target plan. For example, if you want to migrate your service to ``cloud_region``, the command would be:
+
+    avn service update --cloud cloud_region service_to_migrate
+
+You can also follow the progress of your changes with:
```
Comment (laysauchoa) [inline docs/platform/howto/migrate-services.rst:32]:
```suggestion
You can also follow the progress of your changes with::
```

File: docs/platform/howto/migrate-services.rst [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+Your service will be in a *Rebalancing* state, and once complete will be on the new cloud. The service will still be accessible through the process. 
+
+
+Migrate a service using the Aiven client (CLI)
+----------------------------------------------
+
+1. Prepare the command to scale the service.
+
+a. You can retrieve the list of available clouds and regions with:
```
Comment (laysauchoa) [inline docs/platform/howto/migrate-services.rst:24]:
```suggestion
a. You can retrieve the list of available clouds and regions with::
```

File: docs/platform/howto/create_new_service_user.rst [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+    Service user 'testuser' has been created.
+
+
+Create a new service user using the Aiven client (CLI)
+------------------------------------------------------
+
+1. Prepare the command to add the service user to your service.
+
+
+2. Add the ``service_target`` parameter to specify the service for which you want to create a new service user, and the ``service_user`` for the new user. For instance, to add the service user ``service_user`` to the service ``service_target``, you would use this command:
```
Comment (laysauchoa) [inline docs/platform/howto/create_new_service_user.rst:33]:
```suggestion
2. Add the ``service_target`` parameter to specify the service for which you want to create a new service user, and the ``service_user`` for the new user. For instance, to add the service user ``service_user`` to the service ``service_target``, you would use this command::
```

File: docs/platform/howto/scale-services.rst [hunk 8]
```diff
@@ -0,0 +1,34 @@
+Scaling services
+================
+
+When spinning a new Aiven service, you are not tied to a plan. Your services can be adjusted to better match your needs. Services can be moved to a higher or lower plan, and to a different tier—Startup, Business or Premium.
+
+
+Scale a service using the console
+---------------------------------
+
+1. Log in to the Aiven console.
```
Comment (lornajane) [inline docs/platform/howto/scale-services.rst:10]: no need to tell people to log in

File: docs/platform/howto/scale-services.rst [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+1. Log in to the Aiven console. 
+2. Go to your **Services**, and open the service you want to scale.
+3. On the **Overview** tab, scroll down to **Change Plan**. 
+4. Select the new service plan and new tier, if required.
+5. Click **Change**.
+
+Your service will be in a *Rebuilding* state, and once complete will be on the new plan. The service will still be accessible through the process. 
+
+
+Scale a service using the Aiven client (CLI)
```
Comment (lornajane) [inline docs/platform/howto/scale-services.rst:19]: Great example, let's move it to the docs and link to it.

**Review by lornajane — CHANGES_REQUESTED** [review]
First of all, I'm really sorry that we've reviewed this in bits and pieces. We we usually try to take a high level look at the content, and then loop back on spellings and formatting and things. I spotted the IP thing and it's been a headache with new words in the list. So, this isn't how I hoped the process would work, and I'm sorry.

I'm even more sorry that we didn't merge it already because this is valuable stuff!! I added the same comments on every article: let's just have one way to do something, especially these everyday tasks where the CLI one is basically a one-liner. Use the one line command as an additional example (and add some words) in the CLI docs, and then link to it from each of the howtos. Then we can remove subheadings, and also the first step from every howto (we have pretty experienced users so we can make some assumptions about them).

The only other thing I would add is that we try to use realistic values in all the CLI examples. So the variables that hold the service name are `mykafka1` rather than `service_name`, and `aws-us-east1` rather than `cloud-name` and that sort of thing.

File: docs/platform/howto/monitoring-services.rst [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+Review a service using the console
+----------------------------------
+
+1. Log in to the Aiven console. 
+2. Go to your **Services**, and open the service you want to review.
+3. Metrics can be reviewed under the **Metrics** tab, with the choice of spanning the last hour, day, week, month or year.
+4. Logs can be reviewed under the **Logs** tab. Older logs are loaded progressively. To come back to the most recent log entry, click on **Got to most recent message**
+
+
+Review a service using the Aiven client (CLI)
```
[the remark this answers is not in the document]
Comment (frconil) [reply docs/platform/howto/monitoring-services.rst:15]: Agreed, I see we have https://developer.aiven.io/docs/products/flink/concepts/logs-metrics-alerts.html which IMHO should be under platform concepts. I will link back to this.

**Comment by frconil** [comment]
Sorry for the delay and thanks for the thorough feedback.I believed I caught everything, but do let me know if I missed anything.

**Review by lornajane — APPROVED** [review]

**Comment by lornajane** [comment]
This is a raft of great content! Thank you so much for the updates, those CLI examples especially are just beautiful :heart_eyes:

**Pull request merged**